### DISCUSSION TITLE
Website rework

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#115] [Ideas]
# Goal
Have a fresh website for founders drop with new content
(see Miro board for details)

- [x] Intro content
- [x] drop info section
- [ ] artist intro section

# Workflow
![Website-workflow](https://user-images.githubusercontent.com/80257662/133395051-f7b25fc6-aa56-4e08-a1c8-80bac0d18821.jpg)

**Comment by DanM3rcurius** [comment]
Content for the intro for the (founders) drop pages
@evolayak @tamagit80 @jonloone  @wlstylr what do you guys think?

## Who & What's SecretDecks

SecretDecks is a colorful collective of mindful spirits that work towards leaving this plane a better place. Discovering and exploring new tools for self organising communities is the motivator in common. All members share an agreed upon value system, defined in the (Mindfulness Policy)[] that sets the base vibration. 
Our goal is  create a Tarot metaverse on Web 3.0 architecture within a distributed cooperative and freely share the tools we discover and create used to help other cooperatives grow and prosper.
SecretDecks first artist collaboration is with two amazing Italian artists, introducing their original Tarot content to the realm of virtual collectibles.

### How we do stuff
Anyone can go through the onboarding process, members are being rewarded after tasks are done and contribution was validated by peers. Successively each member is attributed (voting) power on a merit basis. A basic polling system is in place and will soon be extended with an blockchain recorded voting process.

## SD Metaverse Apps
+ SD-"DROPS" - NFT launchpad (for virtual Tarot card collectibles)
+ SD-"READINGS" - p2p Tarot reading; to help with reading the book of Thoth  
+ SD-"SOULJOURNAL" - personal meta-journal, to help connecting with your inner Self
+ SD-"SHARE" - p2p collectible trade, to help complete collections
+ SD-"STAKING" - stake your NFT to receive rewards from SD-SPO

**Comment by evolayak** [comment]
Content for the intro for the (founders) drop pages
@evolayak @tamagit80 @jonloone  @wlstylr what do you guys think?

Like it a lot! Wasn't sure how to do a pull request from here or if it's at that stage yet but, I did a proofread and minor edits and have these suggestions (all in the text below):

plane -> dimension  
the motivator in common -> our common motivation
the base vibration -> the base vibration of our work



## Who & What's SecretDecks

SecretDecks is a colourful collective of mindful spirits who work towards leaving this dimension a better place. Discovering and exploring new tools for self organising communities is our common motivation. All members share an agreed upon value system, defined in the (Mindfulness Policy)[] that sets the base vibration of our work.
Our goal is to create a Tarot metaverse on Web 3.0 architecture within a distributed cooperative and freely share the tools we discover and create to help other cooperatives grow and prosper.
SecretDecks' first artist collaboration is with two amazing Italian artists, introducing their original Tarot content to the realm of virtual collectibles.

### How we do stuff
Anyone can go through the onboarding process, members are being rewarded after tasks are done and their contribution is validated by peers. Successively, each member is attributed (voting) power on a merit basis. A basic polling system is in place and will soon be extended with a blockchain recorded voting process.

## SD Metaverse Apps
+ SD-"DROPS" - NFT launchpad (for virtual Tarot card collectibles)
+ SD-"READINGS" - p2p Tarot reading; to help with reading the book of Thoth  
+ SD-"SOULJOURNAL" - personal meta-journal, to help connecting with your inner Self
+ SD-"SHARE" - p2p collectible trade, to help complete collections
+ SD-"STAKING" - stake your NFT to receive rewards from SD-SPO

----

**Comment by DanM3rcurius** [comment]
Perfect edits, thank you!
And you’re right, I should create a doc from this, maybe this will be my first gitbook page!

**Comment by DanM3rcurius** [comment]
I added the content to our Arcana Gitbook.
@stephen-rowan commits to the gitbook should be tracked via SourceCred as well right?
@evolayak if you want to edit sth in the gitbook you can just edit it and then request a merge within the interface. If you can't find it let me know!

https://app.gitbook.com/@secretdecks/s/arcana/introduction/who-and-what-is-secretdecks

**Comment by stephen-rowan** [comment]
> 
> 
> I added the content to our Arcana Gitbook.
> @stephen-rowan commits to the gitbook should be tracked via SourceCred as well right?
> @evolayak if you want to edit sth in the gitbook you can just edit it and then request a merge within the interface. If you can't find it let me know!
> 
> https://app.gitbook.com/@secretdecks/s/arcana/introduction/who-and-what-is-secretdecks

I suppose we could track GitBook contributions -  commits on the back end repo will be by the GitBook Bot and GitBook member. So individual contributions should be tracked automatically. But we need to try some other commits to see how this works.

See https://github.com/SecretDecks/Arcana and my GitBook commit history https://github.com/SecretDecks/Arcana/commits?author=stephen-rowan

**Comment by wlstylr** [comment]
I'm just seeing all of this now 👀 
Everything looks good to me; great work! 💪

**Comment by DanM3rcurius** [comment]
> https://github.com/SecretDecks/Arcana/commits?author=stephen-rowan

Hm ok, thanks for sharing @stephen-rowan! 
Guess we'll see if the gitbookbot collects any SourceCred ^^ And if yes we need to tweak the flow.
Maybe we work from within github, ie create and commit the page / contents there, then import it into Gitbook with the import URL / markdown function. That way the individual contribution definitely gets tracked even if the bot commits the same content in the Gitbook page after.
Does that make sense?

**Comment by evolayak** [comment]
Thanks!

**Comment by stephen-rowan** [comment]
> 
> 
> > https://github.com/SecretDecks/Arcana/commits?author=stephen-rowan
> 
> Hm ok, thanks for sharing @stephen-rowan!
> Guess we'll see if the gitbookbot collects any SourceCred ^^ And if yes we need to tweak the flow.
> Maybe we work from within github, ie create and commit the page / contents there, then import it into Gitbook with the import URL / markdown function. That way the individual contribution definitely gets tracked even if the bot commits the same content in the Gitbook page after.
> Does that make sense?

Maybe. I have not yet experimented much with updating the back-end repo to a GitBook. 

If you do change content at the backend I think it triggers a bot to update the front end, But how this translates to assignees I am not sure. We need to test it.

It might be wise to setup a test GitBook for sourcecred tests. There is sometimes lags with GitBook  (the front end updates not committing to the backend) - which I think is due to the daily action limits on GitHub. A lot sourcecred actions could exceed this limit.

**Comment by DanM3rcurius** [comment]
drop info section done in Miro & Figma.
Waiting for votes from discord for final names of elements.
@evolayak is working on bio and blurb for artists, we'll use back of their decks as first teaser image.

**Comment by wlstylr** [comment]
I've been taking whats in Figma and building out the website through draft pages in the admin section.  I plan to publish as private pages for review, after a little more progress

**Comment by tamagit80** [comment]
ok perfect. then i will fix what's missing.

**Comment by DanM3rcurius** [comment]
visualised the workflow as discussed in meeting yesterday (see #102 )
![Website-workflow](https://user-images.githubusercontent.com/80257662/133395119-d13d3d0a-9178-487a-a815-3e715856f04d.jpg)

**Comment by stephen-rowan** [comment]
> visualised the workflow as discussed in meeting yesterday (see #102 )

Vue migration coincides well with my Distributed Auditability - Continuous Integration Deployment (if it gets funded).

**Comment by tamagit80** [comment]
### Why choosing Github Pages and a JS Framework is the best choice for SD

- No ownership
- Free hosting for a production and a testing environment on github
- The JS app will be deployed on github pages without any cost
- Content of the website will be automatically filled up with specific public repositories
- Easy integration with external sources/systems.
- Merge action will be shared/controlled by all the members
- No need of a centralized DB
- No need of users creation or login, will use github users
- Everything will be front end
- A custom domain can be connected to it
